modules/core/src/main/scala/higherkindness/droste/scatter.scala

```diff
@@ -3,17 +3,27 @@ package higherkindness.droste
 import cats.Functor
 import cats.syntax.functor._
 
+/** Companion object for `Scatter[F[_], A, S]`, which is a type alias of `S => Either[A, F[S]]`
+  */
 object Scatter {
 
+  /** Returns a Scatterer that lifts the input `A` into a `Left` */
```

Comment:
It's worth noting that this scatter function gives you the canonical anamorphism, hence the name!
`gana(Gather.ana) <~> ana`